src/interfaces/interfaces.ts

```diff
@@ -244,11 +244,11 @@ namespace interfaces {
         whenTargetNamed(name: string | number | symbol): BindingOnSyntax<T>;
         whenTargetIsDefault(): BindingOnSyntax<T>;
         whenTargetTagged(tag: string | number | symbol, value: any): BindingOnSyntax<T>;
-        whenInjectedInto(parent: (Function | string)): BindingOnSyntax<T>;
+        whenInjectedInto(parent: (Function | string | Symbol) | (Function | string | Symbol)[]): BindingOnSyntax<T>;
```

Comment:
lower case symbol seems to be the prevailing convention?

Reply:
Thanks for pointing out that error. I will correct tomorrow.  Should I close the pull request and create another ?